### DISCUSSION TITLE
Can I get reply from this site?

### DISCUSSION THREAD

**Discussion by Melodymoo** [consenlabs/webview#11] [General]
Since our problem is more related to our unrevealed app/service, so I want to know if there any email can I sent my issue directly to the developer team?

**Comment by kaichen** [comment]
could you describe more about your question? and here's only discuss about issues you meet w/ this utils lib.

**Comment by unix** [comment]
This issue is security-related, fixing the `Agent` will prevent `Google Auth` from popping up this alert, but we need to discuss further with the security team if this change might affect user security, **I will update comments again later on the progress of the issue.**

----

For another issue, an unexpected jump to the system browser in the Android platform, is due to the use of statements such as `target="_blank", window.open()`. If you specify to open the webite in a new tab on the Android platform, it will jump to the system browser, this behavior is to avoid security issues on the Android platform, please refer to [compatibility documentation](https://imtoken.gitbook.io/developers/products/webview/compatibility#links) to learn more.

**You can first try another option to complete the Google Auth of your application**:

1. Prompting the user to log in to the system's browser, and complete Google Auth in the system's browser.
2. Jump to imToken via DeepLink: `imtokenv2://navigate/DappView?url=https://yourdomain.com`

The DeepLink allows you to jump to the imToken application in the system browser, please refer to [deep-linking documentation](https://imtoken.gitbook.io/developers/products/deep-linking).

**Reply by Melodymoo** [reply]
Hi, any update for Google Auth issue?

**Reply by unix** [reply]
@Melodymoo 
This issue has been fixed in version `2.9.4`.

You can use the Google OAuth in imToken `v2.9.4`, but be aware that not all users have upgraded to the latest version. If necessary, you can [check the version of the app via the SDK](https://imtoken.gitbook.io/developers/products/webview/sdk#getversion) and remind the user to upgrade if the version is too low.
